### PR TITLE
Fix spacing between stats overview and store management cards

### PR DESCRIPTION
**Pull request by jameskoster**
Before:

before

After:

after

### CODE REVIEW

File: client/homescreen/stats-overview/style.scss [hunk 1]
```diff
@@ -4,6 +4,8 @@ $promo-actions-border-top: $core-grey-light-500;
 $promo-actions-border-bottom: $light-gray-tertiary;
 
 .woocommerce-stats-overview {
+	margin-bottom: $gap-large;
```
Comment (joshuatf) [inline client/homescreen/stats-overview/style.scss:7]: Should this spacing be used across all cards on the homescreen?  Maybe this margin should be moved to `.woocommerce-homescreen-card`.

Or better yet, we could apply this to any root item added to a homescreen column:

```scss
.woocommerce-homescreen-column > div {
	margin-bottom: $gap-large;
}
```

Then we can remove component specific styles like the same bottom margin added to `.woocommerce-homescreen .woocommerce-task-dashboard__container`.

**Review by joshuatf — COMMENTED** [review]
Thanks for this fix, Jay!  Added a quick question about what elements this should be applied to and a potential change so these are covered going forward.

File: client/homescreen/stats-overview/style.scss (hunk 1 above)
Comment (jameskoster) [reply client/homescreen/stats-overview/style.scss:7]: yea that would probably be better :)

**Comment by timmyc** [comment]
Going to close/re-open to get CI to do the dance.

**Comment by timmyc** [comment]
and we're back.

**Review by timmyc — APPROVED** [review]
Rebased to get CI happy, tested. all good. thanks Jay!

**Comment by timmyc** [comment]
cherry-picked into `release/1.3.0-rc.1` in 34b8bbc3df7fc6b9369251110031edd781e7e8dd

**Pull request merged**